onnxruntime/core/providers/rocm/tunable/tunable.h

```diff
@@ -100,6 +103,18 @@ class TunableOp {
     tuning_ = false;
   }
 
+  // We might want to do some tricks to the params, e.g., some op will do inplace update to one output buffer. If we
+  // blindly tune over the `params`, there will be accumulated update to that buffer during FindFastest, which is
+  // undesired side effect. In this case, we must prepare a new (proxy) params struct for the tuning so that the side
```

Comment:
nit:
(1) "an" undesired side effect
(2) for the tuning effect can be hide (hidden)  -> to avoid this side effect